bin/node/src/commands.rs

```diff
@@ -84,6 +91,25 @@ fn open_keystore(
     }
 }
 
+fn bootstrap_backup(chain_params: &ChainParams, account_id: &AccountId) {
```

Comment:
I'm not quite sure what this is about – why do we bootstrap the backup dir, as far as I can tell we would have created it at the point of launching the node (or even session)? Is this just so `bootsrap-[chain|node]` fails early _if_ it's run?

Not saying this is a bad thing, just trying to understand.

Reply:
I thought this would be more inlined with our pipeline. Before runing the node we have everything prepared by runing `bootstrap-chain` or `bootstrap-node`. That's where we create `p2p_secret` and so on. Maybe we should create this directory also in binary in case it is not created? Similar thing happens to `p2p_secret`, but it is a behavior of substrate.